src/cpp/fastdds/topic/ContentFilteredTopicImpl.cpp

```diff
@@ -20,15 +20,18 @@
 
 #include <algorithm>
 
-#include <fastdds/dds/core/policy/ParameterTypes.hpp>
-#include <fastrtps/utils/md5.h>
-
 #include <fastdds/core/policy/ParameterList.hpp>
+#include <fastdds/dds/core/policy/ParameterTypes.hpp>
+#include <fastdds/dds/domain/DomainParticipant.hpp>
+#include <fastdds/domain/DomainParticipantImpl.hpp>
```

Comment:
This header is private so it should be after the public ones. However, I think this header is not being used and instead the DomainParticipantQos header should be included.